### PR TITLE
Further style and content adjustments

### LINKED ISSUE #2: Add base Hugo website to theme
Add a base Hugo site with UI elements to showcase a theme. This should aid in developing the theme itself.

### LINKED ISSUE #13: Add some sort of favicon
A favicon should be in the plans, so that the default tab icon isn't used by
every single browser trying to access the page.

Maybe something using the profile picture I have here on GitHub, or something
randomly generated.

### PR DESCRIPTION
**Pull request by d3adb5**
The partial that generates the navigation bar has been refactored, introducing a
new page parameter, `navIndex`, that helps decide where on the bar a link to the
page should end up. The current page is also detected to aid in styling the link
for the active page.

A favicon was added, which fixes #13.

The 'About' page has been removed in favor of adding more content to the home
page, which now features some bit of explanation for what to expect from the
site and how to contact me.

The site's monospace font, `Source Code Pro`, was admittedly quite big compared
to its sans-serif cousing, `Source Sans Pro`, chosen for the body copy. It was
changed to `Inconsolata`, which is looking a lot better.

I think we can say this closes #2, since we now have a basic website to theme,
despite a few things missing that will come soon. The _Lorem ipsum_ page is
staying for now, with its only change being the addition of a code block, which
wasn't there before.

### CODE REVIEW

File: content/lorem.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ date  = 2022-02-05
```
Comment (pijeong) [inline content/lorem.md:4]: Comment regarding line 3: Change date so it is consistent with last page update.

File: content/_index.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
 +++
-title = "d3adb5"
 date  = 2022-03-08
```
Comment (pijeong) [inline content/_index.md:2]: Date should be updated to be consistent with last change.

**Review by pijeong — CHANGES_REQUESTED** [review]

File: content/lorem.md (hunk 1 above)
Comment (d3adb5) [reply content/lorem.md:4]: Dates updated in a378f02.

File: content/_index.md (hunk 2 above)
Comment (d3adb5) [reply content/_index.md]: Dates updated in a378f02.

**Review by pijeong — APPROVED** [review]

**Pull request merged**